apps/native/app/src/screens/family/family-details.tsx

```diff
@@ -20,25 +49,38 @@ const { getNavigationOptions, useNavigationOptions } =
 
 export const FamilyDetailScreen: NavigationFunctionComponent<{
   id: string
-  type: string
+  type: 'bioChild' | 'custodyChild' | 'spouse'
```

Comment:
these types are they accessible in generated code?

Reply:
No unfortunately, I create them [here](https://github.com/island-is/island.is/blob/7b15be87b693f02671f8f660748c8d20a8a9f181/apps/native/app/src/screens/family/family-overview.tsx#L109).

Reply:
No unfortunately not